backend/README.md

```diff
@@ -130,7 +131,7 @@ coverage run ./manage.py test
 
 ## Database diagram
 
-![Database Schema Diagram](docs/fms_database_diagram.png)
+[Database Schema Diagram](https://dbdiagram.io/d/604aa285fcdcb6230b23c0e7)
```

Comment:
If this is on master, maybe it should include the latest tables, such as Sample Kind, Sample Lineage, Protocol, etc. for example?

Reply:
yeah you are right i must be sleep walking to do that on master with the old diagram ... I'll include the new changes

Reply:
Ok updated the diagram to be in sync with master instead ...